README.md

```diff
@@ -1,8 +1,15 @@
+# ClientBook
+
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2021S2-CS2103T-W15-2/tp/actions)
 
+ClientBook is an application for insurance agents to manage client contacts, optimized for use with just a keyboard,
+but still offering a graphical interface for ease of viewing of client contacts. The purpose of ClientBook is to help
+insurance agents who can type fast to accomplish their client management tasks faster than traditional applications.
+
+Get our latest release [here](https://github.com/AY2021S2-CS2103T-W15-2/tp/releases)!
+
 ![Ui](docs/images/Ui.png)
```

Comment:
Will be good if the Ui image can be updated to reflect client contacts that have some policy URLs, meeting details and relevant tags.

Reply:
Updated! Thanks for the catch.